internal/controller/policyserver_controller_deployment.go

```diff
@@ -27,6 +27,7 @@ const (
 	policiesVolumeName               = "policies"
 	sourcesVolumeName                = "sources"
 	verificationConfigVolumeName     = "verification"
+	clientCAVolumeName               = "client-ca-cert"
```

Comment:
Rename it to make it obvious that's the k8s API server CA. Also, always mount the internal Kubewarden CA, that's required to verify the requests issued by `audit-scanner`

Reply:
Also, unrelated with this code line, we have to update the definition of the PolicyServer deployment to consume the `readiness` probe from a different port.

The Policy Server is always going to listen on two different ports:
- One without TLS for the readiness probe
- On with TLS (and eventually mTLS) for all the incoming requests